tests/yam/agama/boot_agama.pm

```diff
@@ -39,6 +39,8 @@ sub prepare_boot_params {
         return @params;
     }
 
+    push @params, 'inst.dud=' . data_url(get_var('INST_DUD')) . ' rd.neednet=1' if (get_var('INST_DUD'));
```

Comment:
you can move this to the end, after network params with a comment like:
`Additional parameters requiring parsing` introducing new block for those (not for testing a feature, because this is a mechanism that would work for different scenarios/features, for example this is a dummy dud but later we need to implement a dud with some particular structure I believe, like in 15).

Reply:
updated.